### DISCUSSION TITLE
is there any reason for why there is no wiki about TMDbHelper.EnableCrop ?

### DISCUSSION THREAD

**Discussion by Nanomani** [jurialmunkey/plugin.video.themoviedb.helper#1249] [Q&A] [answered]
@jurialmunkey 

hi,
I m just curious.
is there any reason for why there is no wiki about `TMDbHelper.EnableCrop` ?

It is not really a problem I use this feature whitout any issue.
All is perfectely documented but not for this feature `TMDbHelper.EnableCrop`.

Maybe I need a new pair of glasses ? :)

Have a good day.
Nicolas.

**Comment by jurialmunkey** [comment, marked as answer]
I just forgot to add it.

**Comment by Nanomani** [comment]
It is a good reason :)
Anyway thanks for the answers and your time.
